src/main/java/org/springframework/data/elasticsearch/annotations/Field.java

```diff
@@ -29,13 +31,30 @@
  * @author Jonathan Yan
  * @author Jakub Vavrik
  * @author Kevin Leturc
+ * @author Peter-Josef Meisch
  */
 @Retention(RetentionPolicy.RUNTIME)
 @Target(ElementType.FIELD)
 @Documented
 @Inherited
 public @interface Field {
 
+	/**
+	 * The name to be used to store the field inside the document. Alias for {@link #name()}.
+	 *
+	 * @return an empty {@link String} by default.
```

Comment:
Not sure if we really need this. IMO we should point out that if empty the property name is used.